src/system/SystemComponent.cpp

```diff
@@ -360,6 +360,7 @@ QString SystemComponent::getNativeShellScript()
 /////////////////////////////////////////////////////////////////////////////////////////
 void SystemComponent::checkForUpdates()
 {
+#ifndef DISABLE_UPDATE_CHECK
```

Comment:
As long as the setting is set correctly, there is no reason to have a #ifndef here.